{{cookiecutter.plugin_name}}/.github/workflows/test_and_deploy.yml

```diff
@@ -61,7 +61,7 @@ jobs:
           PLATFORM: ${{ matrix.platform }}
 
       - name: Coverage
-        uses: codecov/codecov-action@v3
```

Comment:
I cannot agree for this change. codecov-action@v4 Does not support tokenless upload (with exception for PR for open source projects).

Reply:
I didn't know. I reversed the modification.

Reply:
Yeah. They describe it as an easy migration, but we still stuck with this: https://github.com/napari/napari/pull/6701